app/src/main/java/com/globant/example/mentorapp/home/domain/model/EventApiResponseEntity.java

```diff
@@ -0,0 +1,49 @@
+package com.globant.example.mentorapp.home.domain.model;
+
+import java.util.List;
+
+/**
+ * Class to manage otto responses from ApiClient
+ * Created by oscar.amezquita on 13/06/2017.
+ */
+
+public class EventApiResponseEntity {
```

Comment:
Remove setters, declare attributes as final and implement *Builder Pattern*, implement an attribute *T data* so your class is scalable and works for any kind of responses, not only a list of uses.